<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addSunButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>获取阳光</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sunNotDecreaseCheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>阳光不减</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="plantLockHPCheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>植物锁血</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="coolDownCheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>植物0cd</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoCollectCheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>自动收集</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="doomShroomCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>毁灭菇部署</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="potatoMineDeployCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>土豆雷部署</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cobCannoDeployCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>玉米加农部署</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chomperDeployCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>大嘴花部署</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="magnetShroomDeployCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>磁力菇部署</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="plantAtkBulletCheckBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>子弹叠加</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="blockIceSurfaceCheckBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>取消生成冰面</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="blockGravestoneCheckBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>取消生成墓碑</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="blockFogCheckBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>取消生成迷雾</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="backRunnerCheckBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>后台运行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="plantDeployLimitCheckBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>解除种植限制</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="purpleCardLimitCheckBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>解除紫卡限制</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="shroomAwakenCheckBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>蘑菇免唤醒</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionInit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInit">
   <property name="text">
    <string>重置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
